### PR TITLE
Add example + enhance working

### PR DESCRIPTION
**Pull request by FriggaHel**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -70,11 +70,6 @@ workflows:
     when: << pipeline.parameters.run-integration-tests >>
     jobs:
       - integration-test_saucectl
-      - saucectl/saucectl-run:
```
Comment (sweeneyskirt-sl) [inline .circleci/config.yml:73]: You don't need to define this in the yaml anymore?

File: src/@orb.yml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 version: 2.1
 description: |
-  SauceCtl orb allow you to manage simply the task to run with saucectl.
+  Saucectl is a command line interface orchestrates the relationship between
+  your tests in your framework, and the rich parallelization, test history
+  filtering, and analytics of Sauce Labs.
+
+  Usage requires an Sauce Labs account.
```
Comment (sweeneyskirt-sl) [inline src/@orb.yml]:
```suggestion
  saucectl is a command line interface that orchestrates the relationship between
  your tests in your framework, and the rich parallelization, test history
  filtering, and analytics of Sauce Labs.

  Usage requires a Sauce Labs account.
```

File: src/commands/saucectl-run.yml [hunk 3]
```diff
@@ -1,11 +1,23 @@
 description: |
-  This command runs "saucectl run".
+  This command fetches saucectl binary and then execute "saucectl run".
+
+  Note: a Sauce Lab account is required. Username and access key need to be
+  set in order to make saucectl command to succeed. Credentials can be set
+  using env variables SAUCE_USERNAME and SAUCE_ACCESS_KEY or using orb parameters.
```
Comment (sweeneyskirt-sl) [inline src/commands/saucectl-run.yml]:
```suggestion
Note: A Sauce Labs account is required. You must set Username and Access Key credentials either through the env variables SAUCE_USERNAME and SAUCE_ACCESS_KEY or using orb parameters in order to execute the command.
```

File: src/examples/run_on_docker.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+description: |
+  This example illustrate how to execute your tests in Docker through
+  saucectl.
+
+  CircleCI require you to provision a remote docker host in order to be able
+  to execute any docker container.
```
Comment (sweeneyskirt-sl) [inline src/examples/run_on_docker.yml]:
```suggestion
  CircleCI requires you to provision a remote Docker host in order to be able
  to execute any Docker container.
```

File: src/examples/run_on_docker.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+  Note: a Sauce Lab account is required. Username and access key need to be
+  set in order to make saucectl command to succeed. Credentials can be set
+  using env variables SAUCE_USERNAME and SAUCE_ACCESS_KEY or using orb
+  parameters.
```
Comment (sweeneyskirt-sl) [inline src/examples/run_on_docker.yml]:
```suggestion
Note: A Sauce Labs account is required. You must set Username and Access Key credentials either through the env variables SAUCE_USERNAME and SAUCE_ACCESS_KEY or using orb parameters in order to execute the command.
```

File: src/examples/run_on_docker.yml [hunk 6]
```diff
@@ -0,0 +1,29 @@
+description: |
+  This example illustrate how to execute your tests in Docker through
+  saucectl.
```
Comment (sweeneyskirt-sl) [inline src/examples/run_on_docker.yml]:
```suggestion
  This example illustrates how to execute your tests in Docker through
  saucectl.
```

File: src/examples/run_on_sauce.yml [hunk 7]
```diff
@@ -0,0 +1,24 @@
+description: |
+  This example illustrate how to execute your tests in Sauce Labs through
+  saucectl.
```
Comment (sweeneyskirt-sl) [inline src/examples/run_on_sauce.yml]:
```suggestion
  This example illustrates how to execute your tests in Sauce Labs through
  saucectl.
```

File: src/examples/run_on_sauce.yml [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+  Note: a Sauce Lab account is required: username and access key need to be
+  set in order to make saucectl command to succeed. Credentials can be set
+  using env variables SAUCE_USERNAME and SAUCE_ACCESS_KEY or using orb
+  parameters.
```
Comment (sweeneyskirt-sl) [inline src/examples/run_on_sauce.yml]:
```suggestion
Note: A Sauce Labs account is required. You must set Username and Access Key credentials either through the env variables SAUCE_USERNAME and SAUCE_ACCESS_KEY or using orb parameters in order to execute the command.
```

**Review by sweeneyskirt-sl — COMMENTED** [review]
Couple grammar suggestions. Otherwise looks good. :)

File: .circleci/config.yml (hunk 1 above)
Comment (FriggaHel) [reply .circleci/config.yml:73]: This removed element is the `job` (which is composed of `commands`).
Part of that rewrite is to only rely on the command, as there is no benefits of providing a job definition.

**Review by sweeneyskirt-sl — APPROVED** [review]
👍

**Review by KyleTryon — APPROVED** [review]
Thank you for these changes

File: README.md [hunk 9]
```diff
@@ -11,43 +11,31 @@ Both are called `saucectl-run`.
...
-| saucectl-version | Version of saucectl to use. Example: v0.25.1 | `latest` |
+| sauce-username | Sauce Username to use for authentification | |
+| sauce-access-key | Sauce Access Key to use for authentification | |
+| saucectl-version | Version of saucectl to use. Example: v0.68.2 | `latest` |
 | config-file | Configuration file to use with saucectl | `.sauce/config.yml` |
 | working-directory | Working directory to use when running saucectl | `.` | 
 | region | Region to pass to saucectl | |
-| testing-environment | Testing Environment | |
-| suite | Suite to be tested | |
+| select-suite | Suite to be executed | |
```
Comment (alexplischke) [inline README.md]:
```suggestion
| select-suite | Run a particular suite instead of all | |
```

File: README.md [hunk 10]
```diff
@@ -11,43 +11,31 @@ Both are called `saucectl-run`.
 > - SAUCE_USERNAME
 > - SAUCE_ACCESS_KEY
 
+Alternativement, you can use `sauce-username` and `sauce-access-key` parameters of the orb.
```
Comment (alexplischke) [inline README.md]:
```suggestion
Alternatively, you can use the `sauce-username` and `sauce-access-key` parameters of the orb.
```

**Pull request merged**